sdk/devopsinfrastructure/arm-devopsinfrastructure/package.json

```diff
@@ -78,7 +78,7 @@
     "@vitest/browser": "^3.0.3",
     "@vitest/coverage-istanbul": "^3.0.3",
     "dotenv": "^16.0.0",
-    "eslint": "^8.55.0",
+    "eslint": "^9.9.0",
     "playwright": "^1.50.0",
     "tshy": "^2.0.0",
```

Comment:
Can you also remove tshy?  Shouldn't be necessary anymore

Reply:
sure.